tools/Common.CS.props

```diff
@@ -45,7 +45,7 @@
 
   <!-- For our C# projects, we want to pull in the code analyzers -->
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="6.0.0">
+    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="7.0.0">
```

Comment:
I believe the reason that I hadn't upgraded this was because the new package doesn't actually work. I'd need to double check now, but the last couple times I did, the analyzers weren't actually loading.

Reply:
At least on my machine, they work (at least during build time), and not using them causes a warning. I'll revert that though if it turns out to cause issues on other dev environments.